extraConfigCX.py

```diff
@@ -0,0 +1,50 @@
+from clustersConfig import ClustersConfig
+import host
+import coreosBuilder
+from concurrent.futures import ThreadPoolExecutor
+from k8sClient import K8sClient
+from nfs import NFS
+from concurrent.futures import Future
+from typing import Dict
+import sys
+from logger import logger
+from clustersConfig import ExtraConfigArgs
+
+"""
+The "ExtraConfigCX" is used to put the CX in a known good state. This is achieved by
+1) Having a CoreOS Fedora image ready and mounted on NFS. This is needed for loading
+a known good state on each of the workers.
+2) Then SSH-ing into the load CoreOS Fedora image, we can run a pod with all the BF2
+tools available. https://github.com/bn222/dpu-tools/
+3) The scripts will try to update the firmware of the CX with mlxup.
+4) Then the worker node is cold booted. This will also cold boot the CX.
+"""
+
+
+def ExtraConfigCX(cc: ClustersConfig, _: ExtraConfigArgs, futures: Dict[str, Future[None]]) -> None:
+    coreosBuilder.ensure_fcos_exists()
+    logger.info("Updating CX firmware on all workers")
+    lh = host.LocalHost()
+    nfs = NFS(lh, cc.external_port)
+    iso_url = nfs.host_file("/root/iso/fedora-coreos.iso")
```

Comment:
BF-2 requires this because the card itself runs an operating system. This is not the case for cx-5/6/7